test/autotest.sh

```diff
@@ -228,26 +209,20 @@ echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 echo "~~~ NAT TCP TEST ~~~"
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 (./build/test/picoapp.elf --vde pic0:/tmp/pic0.ctl:10.40.0.10:255.255.0.0::: --vde pic1:/tmp/pic1.ctl:10.50.0.10:255.255.0.0: -a natbox:10.50.0.10) &
-pids="$! "
 sleep 2
 (./build/test/picoapp.elf --vde pic0:/tmp/pic1.ctl:10.50.0.8:255.255.0.0::: -a tcpbench:r:6667:) &
-pids+="$! "
 sleep 2
 ./build/test/picoapp.elf --vde pic0:/tmp/pic0.ctl:10.40.0.9:255.255.0.0:10.40.0.10::: -a tcpbench:t:10.50.0.8:6667: || exit 1
-for pid in $pid; do wait $pid || exit 1; done
```

Comment:
Yes, it was on purpose. Those lines shouldn't have been added in the first place. If you look at the PR as a whole, as opposed to that one commit, you'll see it becomes much clearer, those lines simply aren't in there.

If you like, you can try adding these again, but I think you'll get an infinite wait.
Of the 2 situations I described earlier, this is the second one : we spawn a never-ending process, so we need to kill it.